webapp/src/js/bootstrapper/translator.js

```diff
@@ -8,6 +8,13 @@ var TRANSLATIONS = {
       return 'Fetching info (' + x + ' docs)…';
     },
     LOAD_APP: 'Loading app…',
+    PURGE_INIT: 'Checking data…',
+    PURGE_INFO: function(progress) {
+      const n = progress.purged;
+      const percent = Math.floor((progress.processed / progress.total) * 100);
+      return `Cleaned ${n} documents (${percent}% complete)…`;
+    },
+    PURGE_AFTER: 'Optimising…',
```

Comment:
Obviously these two keys need to be translated into all the languages we support. It might be difficult to get this done on short notice so feel free to raise an issue blocking the 3.4.0 release to get the translations for each language.